Reed/Components/BottomSheetView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  BottomSheetView.swift
+//  Reed
+//
+//  Created by Hugo Zhan on 11/7/20.
+//  Copyright © 2020 Roger Luo. All rights reserved.
+//
+
+import SwiftUI
+
+fileprivate enum Constants {
```

Comment:
Nice access modifier detail and declaring constants separately. 💯

Reply:
thanks